src/backend/marsha/core/migrations/0052_migrate_livesession_liveattendance.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+
+from django.db import migrations, models
+
+
+def migrate_live_attendance(apps, schema_editor):
+    """Update existing records to set live_attendance with timestamps
+    keys."""
+    Livesessions = apps.get_model("core", "LiveSession")
+    Livesessions_to_migrate = Livesessions.objects.filter(live_attendance__isnull=False)
+    for livesession in Livesessions_to_migrate:
+        for key in list(livesession.live_attendance):
+            try:
+                datetime.fromtimestamp(int(key))
+            except ValueError as error:
+                try:
+                    new_key = int(int(key) / 1000)
+                    datetime.fromtimestamp(new_key)
+                    livesession.live_attendance[
+                        str(new_key)
+                    ] = livesession.live_attendance.pop(key)
+                except ValueError as error:  # shouldn't happen
```

Comment:
> what type of exception should be raised?

None, you let bubble the `ValueError` exception.

> @lunika wdyt we should handle this?

What should we do if this case happen ? If the solution is to remove the key then IMO we should keep this code. Otherwise we have to handle this manually in the database.